packages/utils/accept-header.ts

```diff
@@ -0,0 +1,134 @@
+export function acceptLanguage(header = '', preferences?: string[]) {
+  return (
+    parse(header, preferences, {
+      type: 'accept-language',
+      prefixMatch: true,
+    })[0] || ''
+  );
+}
+
+interface Selection {
+  pos: number;
+  pref?: number;
+  q: number;
+  token: string;
+}
+
+interface Options {
+  prefixMatch?: boolean;
+  type: 'accept-language';
+}
+
+function parse(raw: string, preferences: string[] | undefined, options: Options) {
+  const lowers = new Map<string, { orig: string; pos: number }>();
```

Comment:
I guess we could declare a type for `lowers` Map, so that we can avoid `// @ts-ignore` on line 120